beads_db/README.md

```diff
@@ -1,19 +0,0 @@
-#clear check sum errors
```

Comment:
it would be better to leave README.ms file
and put a comment about what frameworks module use and how to use it.
what to do if anybody wants to change it.